spki/src/algorithm.rs

```diff
@@ -14,7 +14,7 @@ use der::{Decodable, DecodeValue, Decoder, DerOrd, Encodable, Header, Sequence,
 /// ```
 ///
 /// [RFC 5280 Section 4.1.1.2]: https://tools.ietf.org/html/rfc5280#section-4.1.1.2
-#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+#[derive(Copy, Clone, Debug, Eq, PartialEq, PartialOrd)]
```

Comment:
Might as well add `Ord` at this point

Reply:
Most of the testing was with Ord. It was dropped while preparing the PR as not necessary to satisfy the requirements generated by the partial SignedData impl

Reply:
I just added `Ord`.